### PR TITLE
feat: add CI templates for fluent-operator

### LINKED ISSUE #207: Add CI templates  for fluentbit-operator
Refer to https://github.com/fluent/fluentd/tree/master/.github

/help
/first-good-issue

### PR DESCRIPTION
**Pull request by mangoGoForward**
Signed-off-by: mango <xu.weiKyrie@foxmail.com>

### What this PR does / why we need it:
Add CI templates for fluentbit-operator.

### Which issue(s) this PR fixes:

Fixes #207

### Does this PR introduced a user-facing change?

```release-note
None
```

### Additional documentation, usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by zhu733756** [comment]
Seems that DCO failed,  please commit with a signature.

```
git config --global user.name zhu733756
git config --global user.email <email>
git commit -s -m "xxx"
```

**Comment by mangoGoForward** [comment]
> Seems that DCO failed, please commit with a signature.
> 
> ```
> git config --global user.name <name>
> git config --global user.email <email>
> git commit -s -m "xxx"
> ```

Got, thanks.

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
+name: Bug Report
+description: Create a report with a procedure for reproducing the bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Check [CONTRIBUTING guideline](https://github.com/fluent/fluentd/blob/master/CONTRIBUTING.md) first and here is the list to help us investigate the problem.
+  - type: textarea
```
Comment (zhu733756) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:8]: Replace the link please.

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+name: Feature request
+description: Suggest an idea for this project
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Check [CONTRIBUTING guideline](https://github.com/fluent/fluentd/blob/master/CONTRIBUTING.md) first and here is the list to help us investigate the problem.
```
Comment (zhu733756) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:7]: Replace the link please.

File: .github/ISSUE_TEMPLATE/bug_report.yaml (hunk 1 above)
Comment (mangoGoForward) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:8]: But I didn't found a `CONTRIBUTING` file in this repository could to replace.

Comment (mangoGoForward) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:8]: But I didn't found a `CONTRIBUTING` file in this repository could to replace.

Comment (zhu733756) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:8]: Let's copy one CONTRIBUTING file to this repo :)

Comment (mangoGoForward) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:8]: Whether the link (https://github.com/fluent/fluent-operator#contributing) can be replaced in this place?

Comment (mangoGoForward) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:8]: Whether the link (https://github.com/fluent/fluent-operator#contributing) can be replaced in this place?

Comment (mangoGoForward) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:8]: Ok.

Comment (zhu733756) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:8]: > Whether the link (https://github.com/fluent/fluent-operator#contributing) can be replaced in this place?

Yes

**Comment by zhu733756** [comment]
Please squash the commits after you finished. :)

File: .github/ISSUE_TEMPLATE/bug_report.yaml (hunk 1 above)
Comment (mangoGoForward) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:8]: Done. add a `CONTRIBUTING` file to replace.

**Comment by zhu733756** [comment]
Please rebase and merge the commits into one commit. :)

**Comment by zhu733756** [comment]
Totally /lgtm

**Comment by mangoGoForward** [comment]
Sorry, this branch is already messed up, I don't know how to retrieve.

**Comment by zhu733756** [comment]
```
git reset HEAD~3 # this commit will reset the changes use non hard mode
git stash   # commit the changes to the cache
git reset HEAD~20 --hard # returns the missed commits
git rebase upstream/master # rebase to the fresh codes
git stash pop # your changes backups
```

**Comment by mangoGoForward** [comment]
> ```
> git reset HEAD~3 # this commit will reset the changes use non hard mode
> git stash   # commit the changes to the cache
> git reset HEAD~20 --hard # returns the missed commits
> git rebase upstream/master # rebase to the fresh codes
> git stash pop # your changes backups
> ```

That saved me, thanks.

**Comment by zhu733756** [comment]
The commits are not squashed, please follow the steps do it again :)

**Comment by mangoGoForward** [comment]
> I updated the details.

Is it correct now?

**Comment by zhu733756** [comment]
> > I updated the details.
> 
> Is it correct now?


It has 18 commits.  If it is hard to achieve, can you open another pr?

**Comment by mangoGoForward** [comment]
> > > I updated the details.
> > 
> > 
> > Is it correct now?
> 
> It still has 18 commits. If it is hard to achieve, can you open another pr?

Ok

**Comment by mangoGoForward** [comment]
I will close this pr, and please redirect to https://github.com/fluent/fluent-operator/pull/248

**Pull request closed without merging**